rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -382,6 +384,106 @@ def test_upgrade_v4_v5(globaldb):
+@pytest.mark.parametrize('reload_user_assets', [False])
+def test_upgrade_v5_v6(globaldb):
+    """Test the global DB upgrade from v5 to v6"""
+    # Check the state before upgrading
+    with globaldb.conn.read_ctx() as cursor:
+        # check existence of unique_cache table
+        cursor.execute(
+            'SELECT COUNT(*) FROM sqlite_master WHERE type="table" and name=?',
+            ('unique_cache',),
+        )
+        assert cursor.fetchone()[0] == 0
+        # get number of entries in general_cache before upgrade
+        cursor.execute('SELECT COUNT(*) FROM general_cache')
+        gen_cache_content_before = cursor.fetchone()[0]
+        # check that unique_cache_keys content are still in general_cache
+        gen_cache_unique_key_content = 0
+        for key_part in V5_V6_UPGRADE_UNIQUE_CACHE_KEYS:
+            cursor.execute(
+                'SELECT COUNT(*) FROM general_cache WHERE key LIKE ?',
+                (f'{key_part.serialize()}%',),
+            )
+            gen_cache_unique_key_content += int(cursor.fetchone()[0])
+
+    # execute upgrade
+    with ExitStack() as stack:
+        patch_for_globaldb_upgrade_to(stack, 6)
+        maybe_upgrade_globaldb(
+            connection=globaldb.conn,
+            global_dir=globaldb._data_directory / 'global_data',
+            db_filename=GLOBAL_DB_FILENAME,
+        )
+    assert globaldb.get_setting_value('version', None) == 6
+    with globaldb.conn.read_ctx() as cursor:
+        # check again existence of unique_cache table
+        cursor.execute(
+            'SELECT COUNT(*) FROM sqlite_master WHERE type="table" and name=?',
+            ('unique_cache',),
+        )
+        assert cursor.fetchone()[0] == 1
+        # check that appropriate cache data is transfered
+        for key_part in V5_V6_UPGRADE_UNIQUE_CACHE_KEYS:
+            cursor.execute(
+                'SELECT COUNT(*) FROM unique_cache WHERE key LIKE ?',
+                (f'{key_part.serialize()}%',),
+            )
+            assert cursor.fetchone()[0] > 0
+            cursor.execute(
+                'SELECT COUNT(*) FROM general_cache WHERE key LIKE ?',
+                (f'{key_part.serialize()}%',),
+            )
+            assert cursor.fetchone()[0] == 0
+        unique_cache_content = cursor.execute('SELECT COUNT(*) FROM unique_cache').fetchone()[0]
+        # get number of entries in general_cache after upgrade
+        cursor.execute('SELECT COUNT(*) FROM general_cache')
+        gen_cache_content_after = cursor.fetchone()[0]
+        assert gen_cache_unique_key_content == unique_cache_content
+        assert gen_cache_content_before == gen_cache_content_after + gen_cache_unique_key_content
+
+
+@pytest.mark.parametrize('globaldb_upgrades', [[]])
+@pytest.mark.parametrize('custom_globaldb', ['v5_global.db'])
+@pytest.mark.parametrize('target_globaldb_version', [5])
+@pytest.mark.parametrize('reload_user_assets', [False])
+def test_upgrade_v5_v6_multiple_values_unique_key(globaldb):
```

Comment:
There is no reason to add an entire different test for this. Just add the values and check for also in the previous test.

Reply:
okayy

Reply:
all done!